pydy/viz/tests/test_scene.py

```diff
@@ -207,6 +207,39 @@ def test_generate_scene_dict(self):
 
         assert scene_dict['objects'][viz_frame_id]['radius'] == 0.1
 
+    def test_custom_camera(self):
+
+        camera_frame = self.ref_frame.orientnew('rot', 'body',
+                                                [np.pi / 2.0,
+                                                 np.pi / 2.0,
+                                                 np.pi / 2.0], 'xyz')
+
+        camera_point = self.origin.locatenew('cam_point', 30.0 *
+                                             camera_frame.z)
+
+        camera = OrthoGraphicCamera('my_camera', camera_frame, camera_point)
+
+        scene = Scene(self.ref_frame, self.origin, self.viz_frame,
+                      cameras=[camera])
+
+        camera_id = id(camera)
+
+        scene.generate_simulation_dict(self.sys.states,
+                                       self.sys.constants.keys(),
+                                       self.sys.integrate(),
+                                       self.sys.constants.values())
+        scene_dict = scene.generate_scene_dict()
+
+        expected_orientation_matrix = np.array([0.0, 0.0, 1.0, 0.0,
+                                                0.0, -1.0, 0.0, 0.0,
+                                                1.0, 0.0, 0.0, 0.0,
+                                                30.0, 0.0, 0.0, 1.0])
+
+        assert_allclose(scene_dict['cameras'][camera_id]['init_orientation'],
+                        expected_orientation_matrix, atol=1e-14)
+
+        assert scene_dict['cameras'][camera_id]['type'] == 'OrthoGraphicCamera'
```

Comment:
Right now there are no tests for camera.py, see #212. I just created this one to try to find out where #207 was going wrong. 

Plus the full scene dict is checked already in the other test. So some here will be redundant.

We will need to add full tests for camera.py. I just wrote this one and though we could merge it.